### ISSUE TITLE
[Timebox] Docker tags not in sync with app version

### ISSUE THREAD

**Issue by shodgson** [twentyhq/twenty#8450]
## Bug Description

When pulling and using the docker image [v0.32.3](https://hub.docker.com/layers/twentycrm/twenty/v0.32.3/images/sha256-0e502faa9f046bb36371462a850da5eaa0da6ca00a45043fa0d378337a9cc541?context=explore), the web app states that the version is v0.32.0 
![image](https://github.com/user-attachments/assets/b57796fb-3676-4f7b-8739-18619e525203)

## Expected behavior

twentycrm/twenty:v0.32.3 should use the v0.32.3 app.

**Comment by FelixMalfait** [comment]
@charlesBochet what do you think about updating twenty-infra and the version source so that the source of version is pulled from the docker image instead of manually upgrading the package.json?

We could introduce a `versionInfo.json`  in `.gitignore`


***Updated dockerfile***

```

   ARG GIT_COMMIT
   ARG GIT_TAG
   ARG BUILD_DATE

   RUN echo '{' > /app/version_info.json && \
       echo '  "commit": "'${GIT_COMMIT}'",' >> /app/version_info.json && \
       echo '  "tag": "'${GIT_TAG}'",' >> /app/version_info.json && \
       echo '  "build_date": "'${BUILD_DATE}'",' >> /app/version_info.json && \
       echo '}' >> /app/version_info.json

   COPY . /app

```

***Context to pass during build step***
```
# Get the latest commit hash
   GIT_COMMIT=$(git rev-parse HEAD)

   # Get the latest tag
   GIT_TAG=$(git describe --tags --abbrev=0)

   # Get the build date
   BUILD_DATE=$(date -u +"%Y-%m-%dT%H:%M:%SZ")

   # Build the Docker image with all the information
   docker build \
     --build-arg GIT_COMMIT=$GIT_COMMIT \
     --build-arg GIT_TAG=$GIT_TAG \
     --build-arg BUILD_DATE=$BUILD_DATE \
     -t your-image-name .     
```

**Comment by FelixMalfait** [comment]
TODO when we tackle this: look at `SENTRY_RELEASE` as it seems to play a similar role

**Comment by prastoin** [comment]
@FelixMalfait Unless I'm mistaken the related bug has been fixed a while ago ?
Created a new issue regarding the workspace versioning along upgrade commands that will cover the version retrieval topic
Please let me know

ps: Feel free to reopen !

**Comment by FelixMalfait** [comment]
@prastoin yes the goal of this issue was to tackle my above comment: https://github.com/twentyhq/twenty/issues/8450#issuecomment-2469738338 ; could you please link the new issue you created? Thanks!

**Comment by prastoin** [comment]
@FelixMalfait  Ok !
Unless I'm mistaken I've already linked the issues but there you go ! https://github.com/twentyhq/core-team-issues/issues/487
I will handle this or at least re-open the question in it ! instinctively I would say that we should always version our package.json along the code source itself, we can still do both
Lets discuss this !

**Comment by FelixMalfait** [comment]
@prastoin sorry I reply through mail notifications so I didn't see the edited version

**Comment by prastoin** [comment]
@Bonapara No worries never thought about it before :,)
